src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-address/veteran-address.js

```diff
@@ -36,5 +41,26 @@ export const uiSchema = {
       },
     },
     emailAddress: emailUI(),
+    'view:confirmEmail': {
+      'ui:required': formData =>
+        formData.veteranContactInformation.emailAddress !== undefined,
+      'ui:validations': [
+        (errors, fieldData, formData) => {
+          if (
+            formData?.veteranContactInformation?.emailAddress !==
+            formData?.veteranContactInformation?.['view:confirmEmail']
+          ) {
```

Comment:
I wonder if there is a way we can create a reusable widget that renders both the email and the confirm email addresses? Seems like this will be a thing moving forward. What do you think?

Reply:
Love the idea. Would be worth writing a ticket for and working on 💯